src/interfaces/StoneHandlerInterface.ts

```diff
@@ -0,0 +1,46 @@
+import { StoneConfig } from "../common";
+import { AudioPlayer, TimerTicking, Tutorial } from "../components";
```

Comment:
Utilize aliasing and index

Reply:
1. @bernhardccodev thanks i fixed on my latest commit.